modules/virt-configuring-vm-real-time.adoc

```diff
@@ -49,56 +50,144 @@ spec:
 ----
 $ virtctl console <vm_name>
 ----
-
-.. Configure huge pages by using the GRUB boot loader command-line interface. In the following example, 8 1G huge pages are specified.
++
+[NOTE]
+====
+The following commands should all run as the root user.
+====
+.. Disable the `irqbalance` service by using the following command:
 +
 [source,terminal]
 ----
-$ grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=8"
+# systemctl disable irqbalance && systemctl stop irqbalance
 ----
-
-.. To achieve low-latency tuning by using the `cpu-partitioning` profile in the TuneD application, run the following commands:
+.. Enable the realtime and nvf repos:
++
+[source,terminal]
+----
+# subscription-manager repos --enable rhel-9-for-x86_64-rt-rpms --enable rhel-9-for-x86_64-nfv-rpms
+----
+.. Install the necessary packages by running the following command:
 +
 [source,terminal]
 ----
-$ dnf install -y tuned-profiles-cpu-partitioning
+# dnf install tuned cloud-init
 ----
+.. To achieve low-latency tuning by using the `realtime-virtual-guest` profile in the TuneD application, run the following commands:
 +
 [source,terminal]
 ----
-$ echo isolated_cores=2-9 > /etc/tuned/cpu-partitioning-variables.conf
+# dnf install kernel-rt realtime-tests tuned-profiles-realtime
 ----
-The first two CPUs (0 and 1) are set aside for house keeping tasks and the rest are isolated for the real-time application.
 +
 [source,terminal]
 ----
-$ tuned-adm profile cpu-partitioning
+# dnf install tuned-profiles-nfv-guest
 ----
+.. Edit `/etc/tuned/realtime-virtual-guest-variables.conf`:
++
+[source,conf]
+----
+#
+# Variable settings below override the definitions from the
+# /etc/tuned/realtime-variables.conf file.
+
+#
+# Core isolation
+#
+# The 'isolated_cores=' variable below controls which cores should be
+# isolated. By default we reserve 1 core per socket for housekeeping
+# and isolate the rest. But you can isolate any range as shown in the
+# examples below. Just remember to keep only one isolated_cores= line.
+#
+# Examples:
+# isolated_cores=2,4-7
+# isolated_cores=2-23
+#
+# Reserve 1 core per socket for housekeeping, isolate the rest.
+# Change this for a core list or range as shown above.
+isolated_cores=2-3 <1>
 
+#
+# Uncomment the 'isolate_managed_irq=Y' bellow if you want to move kernel
+# managed IRQs out of isolated cores. Note that this requires kernel
+# support. Please only specify this parameter if you are sure that the
+# kernel supports it.
+#
+isolate_managed_irq=Y <2>
+
+#
+# Set the desired combined queue count value using the parameter provided
+# below. Ideally this should be set to the number of housekeeping CPUs i.e.,
+# in the example given below it is assumed that the system has 4 housekeeping
+# (non-isolated) CPUs.
+#
+# netdev_queue_count=4
+----
+<1> The first two CPUs (0 and 1) are set aside for house keeping tasks and the rest are isolated for the real-time application.
+<2> isolate_managed_irq should be set to `Y`
```

Comment:
```suggestion
<2> Uncomment the `isolate_managed_irq=`Y` line to move kernel-managed interrupt requests out of isolated cores.
```

Reply:
This line is not commented by default.
Is there an alternative phrasing we could use?

Reply:
Maybe change it to:
```suggestion
<2> Set the `isolate_managed_irq` parameter to `Y` to move kernel-managed interrupt requests out of isolated cores.
```

Reply:
Done.